src/Eppie.CLI/Eppie.CLI/Resources/Program.resx

```diff
@@ -136,11 +136,11 @@
     <comment>Description of the 'exit' command in the help.</comment>
   </data>
   <data name="Menu.InitDescription" xml:space="preserve">
-    <value>Initialize the application and create a new account.</value>
+    <value>Initialize the application and create a master account.</value>
     <comment>Description of the 'init' command in the help.</comment>
   </data>
   <data name="Menu.OpenDescription" xml:space="preserve">
-    <value>Open an existing account.</value>
+    <value>Open an existing master account.</value>
```

Comment:
Maybe we should use 'Eppie account' here?

Reply:
done https://github.com/Eppie-io/Eppie-CLI/pull/72/commits/bc4060b2ce9e2466e8b6de040717f032ca0a0492